test/e2e/arktos/multi_tenancy/test_suites/crd/test_per_tenant_crd.yaml

```diff
@@ -0,0 +1,138 @@
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ Per-Tenant CRD Tests ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This test suite verifies the per-tenant CRD works in Arktos
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+######################################################
+# test variables
+######################################################
+Variables:
+  first_tenant: random_8
+  second_tenant: random_8
+
+###########################################################################################################
+# test setup
+###########################################################################################################
+Tests:
+  - BeforeTestMessage: Starting test setup ...
+    Command: ${kubectl} create tenant ${first_tenant}
+    OutputShouldContain: 
+    - "\ntenant/${first_tenant} created\n"
+
+  - Command: ${setup_client_script} ${first_tenant} admin
+
+  - Command: ${kubectl} create tenant ${second_tenant}
+    OutputShouldContain: 
+    - "\ntenant/${second_tenant} created\n"
+
+  - Command: ${setup_client_script} ${second_tenant} admin
+
+############################################
+# Verifying different tenants can create their CRDs independently...
+############################################
+  - BeforeTestMessage: Verifying different tenants can create their CRDs independently...
+    Command: ${kubectl} get crds --context ${first_tenant}-admin-context
+    OutputShouldBe: "No resources found.\n"
+
+  - Command: ${kubectl} apply -f ${test_data_dir}/regular_crd.yaml --context ${first_tenant}-admin-context
+    OutputShouldBe: "customresourcedefinition.apiextensions.k8s.io/neversharingsweetdreams.fantasy.wonderland.com created\n"
+
+  - Command: ${kubectl} get crds --context ${first_tenant}-admin-context
+    OutputShouldContain:
+    - "neversharingsweetdreams.fantasy.wonderland.com"
+
+  - Command: ${kubectl} api-resources --context ${first_tenant}-admin-context
+    OutputShouldContain:
+    - "neversharingsweetdreams"
```

Comment:
why 3 different ones?

Reply:
As discussed offline, for some commands without structured outputs ( such as api-resources), I try to make sure that the critical information are displayed by checking multiple pieces of info are displayed. The three pieces here are he singular name, plural name and the resource group, respectively.